.evergreen/config.yml.in

```diff
@@ -167,12 +167,26 @@ functions:
           ${PREPARE_SHELL}
           # Disable xtrace (just in case it was accidentally set).
           set +x
-          MONGODB_URI="${MONGODB_URI}" \
-          MONGODB_API_VERSION="${MONGODB_API_VERSION}" \
-          AUTH="auth" SSL="ssl" SERVERLESS=1 \
-          SERVERLESS_ATLAS_USER="${SERVERLESS_ATLAS_USER}" \
-          SERVERLESS_ATLAS_PASSWORD="${SERVERLESS_ATLAS_PASSWORD}" \
-            bash ${PROJECT_DIRECTORY}/.evergreen/run-serverless-tests.sh
+
+          export MONGODB_API_VERSION="${MONGODB_API_VERSION}"
+          export AUTH="auth"
+          export SSL="ssl"
+          export SERVERLESS="1"
+          export SERVERLESS_ATLAS_USER="${SERVERLESS_ATLAS_USER}"
+          export SERVERLESS_ATLAS_PASSWORD="${SERVERLESS_ATLAS_PASSWORD}"
+
+          export SINGLE_ATLASPROXY_SERVERLESS_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}"
+          export MULTI_ATLASPROXY_SERVERLESS_URI="${MULTI_ATLASPROXY_SERVERLESS_URI}"
+
+          export MONGODB_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}"
+          export SINGLE_MONGOS_LB_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}"
+
+          # Setting MULTI_MONGOS to the SINGLE_ATLAS is intentional
+          # LB tests pick one host out of the comma separated list
```

Comment:
why do we even need the MULTI_MONGOS_LB_URI if it only ever runs against one?

Reply:
Our LB tests are written to make use the MULTI/SINGLE_MONGOS_LB_URI variable. So they don't run if both aren't defined.

In general why does it use only one host, maybe @durran can help me recall the details?

Reply:
The unified tests will define whether to use multiple mongoses or not, which in turn in LB mode will go through this code path: https://github.com/mongodb/node-mongodb-native/blob/4.1/test/tools/runner/config.js#L247 . So this seems like an issue introduced post LB implementation?